Dockerfile

```diff
@@ -0,0 +1,45 @@
+#  Copyright (C) 2018-2019 LEIDOS.
+# 
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy of
+#  the License at
+# 
+#  http://www.apache.org/licenses/LICENSE-2.0
+# 
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+
+FROM usdotfhwastol/carma-base:2.8.3 as setup
+
+RUN sudo apt-get update && sudo apt-get install -y ros-kinetic-jsk-recognition-msgs
+
+RUN mkdir ~/src
+COPY --chown=carma . /home/carma/src/
+RUN chmod +x ~/src/docker/checkout.sh && chown carma ~/src/docker/checkout.sh 
```

Comment:
If you `chmod +x` these scripts and commit them that way this step is unnecessary. Same for the `chown` part, since that `COPY --chown=carma` directive already sets the ownership.